rust/release/src/main.rs

```diff
@@ -304,18 +304,26 @@ Overview:"#
                 println!("-------------------------\n");
                 println!("Branch to create {}", branch_name);
             } else {
-                // This pathing abuses the fact that we control where this is run from: CI
-                // and in CI this is *usually* run in the rust/release path
-                let ax_cargo = PathBuf::from("../actyx/ax/Cargo.toml").canonicalize()?;
-                let ax_core_cargo = PathBuf::from("../actyx/ax-core/Cargo.toml").canonicalize()?;
+                // We expect the binary to not have been moved outside of the target folder
+                let cwd = current_exe()?;
+                let actyx_folder = cwd
+                    .ancestors()
+                    .nth(4)
+                    .ok_or(anyhow!("failed to get the current directory parent"))?;
+                let ax_cargo = actyx_folder.join(PathBuf::from("actyx/ax/Cargo.toml")).canonicalize()?;
+                let ax_core_cargo = actyx_folder
+                    .join(PathBuf::from("actyx/ax-core/Cargo.toml"))
+                    .canonicalize()?;
+
                 for (product, v) in new_versions {
                     let new_version = v.new_version.unwrap();
                     match product {
                         Product::Ax => {
                             eprint!("0.1) Writing new version to \"{}\" ... ", ax_cargo.display());
                             update_package_version(&ax_cargo, &new_version)?;
+                            let version_rs = PathBuf::from("../actyx/ax-core/node/version.rs").canonicalize()?;
```

Comment:
It might be better to also use a path relative from `actyx_folder` (or `rust_folder`) here.